packages/infrastructure/kube_pg_cluster/main.tf

```diff
@@ -367,12 +369,12 @@ resource "kubernetes_manifest" "postgres_cluster" {
             shared_preload_libraries   = ""
             ssl_max_protocol_version   = "TLSv1.3"
             ssl_min_protocol_version   = "TLSv1.3"
-            wal_keep_size              = "2GB"
+            wal_keep_size              = "${var.pg_wal_keep_size_gb}GB"
             wal_level                  = "logical"
             wal_log_hints              = "on"
             wal_receiver_timeout       = "5s"
             wal_sender_timeout         = "5s"
-            max_slot_wal_keep_size     = "10GB"
+            max_slot_wal_keep_size     = "${var.pg_max_slot_wal_keep_size_gb}GB"
```

Comment:
If you are going to make this configurable, then you need to have this feed into the disk storage calculations.

Reply:
@fullykubed Not knoweldgeable enough here. Feels like we have set an upper bound storage limit of 100gb. Given our default initial storage size of 10, the stroage limit is already at 100. 

Maybe just add a max of 100gb to the max_slot_wal_size?

Reply:
Added https://github.com/Panfactum/stack/blob/a67ef6f20cc2b70fa2127875fadeb455d618e1f6/packages/infrastructure/kube_pg_cluster/vars.tf#L575-L578